ios/browser/api/ads/brave_ads.mm

```diff
@@ -199,8 +198,7 @@ - (BOOL)isServiceRunning {
 }
 
 - (BOOL)shouldShowSponsoredImagesAndVideosSetting {
-  const std::string country_code =
-      brave_l10n::GetCountryCode(self->_localStatePrefService);
+  const std::string country_code = brave_l10n::GetDefaultISOCountryCodeString();
```

Comment:
VideoNTT has been replaced by RichNTT. However, it’s correct that we use the device region to determine whether to show the images and videos option in settings.